fiftyone/core/session.py

```diff
@@ -340,14 +341,7 @@ def __init__(
                     "Cannot open a Desktop App instance from a Colab notebook"
                 )
 
-            try:
-                import fiftyone.desktop  # pylint: disable=unused-import
```

Comment:
@benjaminpkane in the current implementation, I think this is allowing a release desktop package to be used when running a source install. I don't see much use for that because we can't guarantee that any app build will work with the bleeding edge of any branch.

Should we just update so `_import_desktop()` is never called if `focn.DEV_INSTALL == True`?

Reply:
Yes, ok